adsrefpipe/tasks.py

```diff
@@ -0,0 +1,115 @@
+from adsrefpipe import app as app_module
+from kombu import Queue
+
+import os
+
+from adsputils import load_config
+
+import adsrefpipe.utils as utils
+from adsrefpipe.xmlparsers.handler import verify
+
+proj_home = os.path.realpath(os.path.join(os.path.dirname(__file__), '../'))
+config = load_config(proj_home=proj_home)
+app = app_module.ADSReferencePipelineCelery('reference-pipeline',
+                                            proj_home=proj_home,
+                                            local_config=globals().get('local_config', {}),
+                                            backend=config.get('REDIS_BACKEND'))
+
+app.conf.CELERY_QUEUES = (
+    Queue('process_references', app.exchange, routing_key='process_references'),
```

Comment:
`process_references` is the main task; the other task is just chopping the message into a smaller task and sending it to the main task -- but that should be done by a client; i.e. `run.py` or whatever is submitting data for processing

let's keep it simple

Reply:
You want me to go back to being a one query pipeline, lolol. 
Let me think about this, one is a full reference file, the other is bits and pieces from multiple reference files, reprocessed, and they are being distinguished.
let me think if they can be combined.
thank you so much man.